### PR TITLE
Load modification modification dialog

### PR DESCRIPTION
**Pull request by niconoir**
A new network modification dialog to create a load modification modification.

### CODE REVIEW

File: src/components/dialogs/load-modification-dialog.js [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+    filledTextField,
+    gridItem,
+    GridSection,
+    ReactivePowerAdornment,
+} from './dialogUtils';
+
+import EquipmentSearchDialog from './equipment-search-dialog';
+import { useFormSearchCopy } from './form-search-copy-hook';
+
+const LOAD_TYPES = [
```
Comment (FranckLecuyer) [inline src/components/dialogs/load-modification-dialog.js:42]: duplicated from load-creation-dialog

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/components/dialogs/load-modification-dialog.js [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+const getId = (e) => e?.id;
+
+/**
+ * Dialog to modify a load in the network
+ * @param {Boolean} open Is the dialog open ?
+ * @param {EventListener} onClose Event to close the dialog
+ * @param voltageLevelOptions : the network voltageLevels available
+ * @param selectedNodeUuid : the currently selected tree node
+ * @param workingNodeUuid : the node we are currently working on
+ * @param editData the data to edit
```
[the remark this answers is not in the document]
Comment (niconoir) [reply src/components/dialogs/load-modification-dialog.js:50]: Parameter description added

File: src/components/dialogs/load-modification-dialog.js [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+    );
+};
+
+LoadModificationDialog.propTypes = {
+    editData: PropTypes.object,
+    open: PropTypes.bool.isRequired,
+    onClose: PropTypes.func.isRequired,
+    voltageLevelOptions: PropTypes.arrayOf(PropTypes.object),
+    selectedNodeUuid: PropTypes.string,
+    workingNodeUuid: PropTypes.string,
```
[the remark this answers is not in the document]
Comment (niconoir) [reply src/components/dialogs/load-modification-dialog.js:250]: equipmentOptions prop type added

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/utils/rest-api.js [hunk 4]
```diff
@@ -1017,6 +1017,56 @@ export function createLoad(
...
+    studyUuid,
+    selectedNodeUuid,
+    id,
+    name,
+    loadType,
+    activePower,
+    reactivePower,
+    voltageLevelId,
+    busOrBusbarSectionId,
+    isUpdate = false,
```
[the remark this answers is not in the document]
Comment (niconoir) [reply src/utils/rest-api.js]: Removed

**Review by FranckLecuyer — APPROVED** [review]

**Review by jonenst — APPROVED** [review]

**Pull request merged**